data/mods/Xedra_Evolved/itemgroups/spell_artifacts/classless.json

```
@@ -0,0 +1,117 @@
+[
+  {
+    "type": "item_group",
+    "id": "xe_classless_spells_low",
+    "//": "Overall group for classless spells, if you don't need a precise tier items. Groups are used in mapgen",
+    "subtype": "distribution",
+    "entries": [
+      { "group": "xe_classless_spell_group0", "prob": 100 },
+      { "group": "xe_classless_spell_group1", "prob": 50 },
+      { "group": "xe_classless_spell_group2", "prob": 25 },
+      { "group": "xe_classless_spell_group3", "prob": 12 },
+      { "group": "xe_classless_spell_group4", "prob": 6 }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "xe_classless_spells_mid",
+    "//": "Same as before, but with higher tier spells",
+    "subtype": "distribution",
+    "entries": [
+      { "group": "xe_classless_spell_group0", "prob": 25 },
+      { "group": "xe_classless_spell_group1", "prob": 50 },
+      { "group": "xe_classless_spell_group2", "prob": 100 },
+      { "group": "xe_classless_spell_group3", "prob": 50 },
+      { "group": "xe_classless_spell_group4", "prob": 25 }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "xe_classless_spells_high",
+    "//": "Same as before, but with higher tier spells",
+    "subtype": "distribution",
+    "entries": [
+      { "group": "xe_classless_spell_group0", "prob": 6 },
+      { "group": "xe_classless_spell_group1", "prob": 12 },
+      { "group": "xe_classless_spell_group2", "prob": 25 },
+      { "group": "xe_classless_spell_group3", "prob": 100 },
+      { "group": "xe_classless_spell_group4", "prob": 50 }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "xe_classless_spell_group0",
+    "//": "Each group contain artifacts of his tier, plus spells of +1 and -1 tier with lower probability. Groups are used in mapgen",
```

```suggestion
    "//": "Tier 0 spell artifacts, plus spells of first tier with lower probability. Groups are used in mapgen",
```
No negative tier artifacts for you :p